### DISCUSSION TITLE
Small moment of praise 🙌

### DISCUSSION THREAD

**Discussion by cwanja** [teslamate-org/teslamate#4322] [General]
Honestly, hats off to the entire development and community team for swarming to resolve a recent issue (plus make the previous releases). Project remains alive and well. 

I know I am going to miss someone, and for that - forgive me. 
@JakobLichterfeld @swiffer @brianmay @jheredianet @DrMichael @Dulanic @jlestel @tobiasehlert

**Comment by JakobLichterfeld** [comment]
Thanks a lot. It takes a village, and obviously you forgot @cwanja 👏🏻

**Comment by giterdone509** [comment]
Thank you all for everything you do! Love everything about TeslaMate

**Comment by jheredianet** [comment]
Thanks a lot @cwanja  for the mention. I've been suffering from health problems these last 6 weeks which have not allowed me to be as active as I would have liked, but I'm regaining strength to continue contributing and doing my bit for this project. Don't forget to heat the road and save your data ;-)

Cheers!!!

**Reply by swiffer** [reply]
take the time needed, wishing you the best.

**Reply by JakobLichterfeld** [reply]
It's good to hear that your health is on the mend again!

**Reply by cwanja** [reply]
Send lots of positive thoughts and improvised health @jheredianet! Sounds like you are on a positive path, hope it stays that way. Take your recovery serious.

**Comment by jlestel** [comment]
@cwanja thanks for the mention 👍  glad to be able to help the wonderful team that supports Teslamate.
